p2p/sender.go

```diff
@@ -288,13 +301,11 @@ func (w legacyReadWriter) WriteMsg(m proto.Message) error {
 func (w legacyReadWriter) ReadMsg(m proto.Message) error {
 	b, err := io.ReadAll(w.stream)
 	if err != nil {
-		return errors.Wrap(err, "read response")
-	} else if len(b) == 0 {
-		return errors.New("peer errored, no response")
+		return errors.Wrap(err, "read proto")
 	}
```

Comment:
@dB2510 this was the zero length check that was erroring, it has been removed.

Reply:
Got it!